util/tracing/span.go

```diff
@@ -0,0 +1,45 @@
+package tracing
+
+import (
+	"os"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+/*
+	Poor Mans OpenTracing.
+
+	Standardizes logging of operation duration.
+*/
+
+var enabled = false
+var logger = log.New()
+
+func init() {
+	enabled = os.Getenv("ARGOCD_TRACING_ENABLED") == "1"
```

Comment:
Trying to find a way to avoid introducing one more env variable. We have a lot of them already. Two options:

1. Log traces with debug level so it will be enabled with `--loglevel debug`
2. Use glog instead of logrus here (glog is already indirectly imported). In this case we could use `--gloglevel 6`